### DISCUSSION TITLE
Better PhD Flopper Icon

### DISCUSSION THREAD

**Discussion by PHDFLOPPERGUY56** [nzp-team/nzportable#754] [Other Content Releases]
I thought the PhD icon looked a little goofy so I went into photoshop and fixed it
[gfx.zip](https://github.com/nzp-team/nzportable/files/15478332/gfx.zip)

**Comment by MotoLegacy** [comment]
I agree this is an improvement. Feel free to make a Pull Request to the asset repository to make this an official change.

**Reply by PHDFLOPPERGUY56** [reply]
I noticed it looks like it didn't end up being transparent for some reason but I fixed it and it looks like it works now so i can definetly submit that
